CHANGELOG.md

```diff
@@ -3,6 +3,64 @@
+* **wallet:** Create react ui scaffold with ses ([#3879](https://github.com/Agoric/agoric-sdk/issues/3879)) ([089c876](https://github.com/Agoric/agoric-sdk/commit/089c876d801efc1ede76b3011a1301384aace77f))
+* **wallet-backend:** add `walletAdmin.getClockNotifier()` ([2902ec3](https://github.com/Agoric/agoric-sdk/commit/2902ec38f6110a5582f6906695b77911f4d43180))
+* **wallet-backend:** attach timestamp and sequence metadata ([9e02962](https://github.com/Agoric/agoric-sdk/commit/9e02962b7c08d56ea1fe72970f3309998b734767))
+* **wallet-connection:** Add getAdminBootstrap functionality to wallet connection ([#3909](https://github.com/Agoric/agoric-sdk/issues/3909)) ([4c56367](https://github.com/Agoric/agoric-sdk/commit/4c563672836edaf92a65fb7829b0a189f7e4ce53))
+* **wallet-connection:** create new cross-framework UI ([905bf6c](https://github.com/Agoric/agoric-sdk/commit/905bf6ccef614b2b8b8d782810b252df9df1abcc))
+* **wallet-connection:** handle dapp approval states ([32b7772](https://github.com/Agoric/agoric-sdk/commit/32b7772ed33ed512ed598bbfc5dcea16ed36a705))
+* Thread URL and Base64 endowments ([b52269d](https://github.com/Agoric/agoric-sdk/commit/b52269d58be665baf45bbb38ace57ca741e5ae4c))
+
+
+### Bug Fixes
+
+* "yarn lint" should not fix ([#3943](https://github.com/Agoric/agoric-sdk/issues/3943)) ([0aca432](https://github.com/Agoric/agoric-sdk/commit/0aca432ed7a3f33eed4575b36dc3fa9e023b445f))
+* adapt timers to async iterables ([#3949](https://github.com/Agoric/agoric-sdk/issues/3949)) ([9739127](https://github.com/Agoric/agoric-sdk/commit/9739127262e9fac48757094a4d2d9f3f35f4bfc5))
+* address review comments ([8af3e15](https://github.com/Agoric/agoric-sdk/commit/8af3e1547b4df32c604f6b628a62bff230666166))
+* bugfixes, comments ([30dbeaa](https://github.com/Agoric/agoric-sdk/commit/30dbeaa7ef64f2c2dee9ddeb0a8c3929a611c21e))
+* document copyRecord guarantees ([#3955](https://github.com/Agoric/agoric-sdk/issues/3955)) ([f4a0ba1](https://github.com/Agoric/agoric-sdk/commit/f4a0ba113dba913c33c37043e700825f3512cf73))
+* lien accounts must proxy all account methods ([db79c42](https://github.com/Agoric/agoric-sdk/commit/db79c42398195a09e8b3953dad35224f0943752b))
+* **eslint-config:** adapt new JSDoc rules ([91fd093](https://github.com/Agoric/agoric-sdk/commit/91fd093bf95f80e19cde520c920b89c50dbf9782))
+* **governance:** export buildParamManager from index.js ([#3952](https://github.com/Agoric/agoric-sdk/issues/3952)) ([868964e](https://github.com/Agoric/agoric-sdk/commit/868964e09cac570cceda4617fd0723a0a64d1841))
+* don't use manual key prefix ([50a881b](https://github.com/Agoric/agoric-sdk/commit/50a881be4971cd5c006867daca66eb6138276492))
+* **SwingSet:** Adjust SES change detectors ([3efb36e](https://github.com/Agoric/agoric-sdk/commit/3efb36eb48521aeb9479f27bd691be485ecda234))
+* **transform-meter:** Accommodate SES module proxies in meter ([9f7d456](https://github.com/Agoric/agoric-sdk/commit/9f7d45602c9d3c0a25729f97c2be8004230cb028))
```

Comment:
boring... we're still maintaining transform-meter

(boring = interest on technical debt. do we have an issue with the **technical-debt** label for this? I don't suppose it's cost-effective to go searching just now...)